apps/researcher/src/lib/community.test.ts

```diff
@@ -0,0 +1,161 @@
+import {describe, expect, it} from '@jest/globals';
+import {clerkClient, auth} from '@clerk/nextjs';
+import {getCommunity, getMemberships, isAdminOf, Membership} from './community';
+
+jest.mock('@clerk/nextjs', () => ({
+  auth: jest.fn().mockImplementation(() => ({
+    userId: 'me',
+  })),
+  clerkClient: {
+    organizations: {
+      getOrganization: jest.fn(),
+      getOrganizationMembershipList: jest.fn(),
+    },
+  },
+}));
+
+describe('getCommunity', () => {
+  it('should return a community', async () => {
+    (clerkClient.organizations.getOrganization as jest.Mock).mockImplementation(
+      () => ({
+        id: 'community1',
+      })
+    );
+
+    const {community, error} = await getCommunity('community1');
+
+    expect(community).toEqual({id: 'community1'});
+    expect(error).toBeUndefined();
+  });
+
+  it('should return an error', async () => {
+    (clerkClient.organizations.getOrganization as jest.Mock).mockImplementation(
+      () => {
+        throw {status: 404};
```

Comment:
For clarity (and typing): would it be possible to throw an (instance of) `Error` instead?

Reply:
I tried, but I could not simulate a Clerk error with an instance of an Error. Maybe I needed to do something differently. But for now, I have removed the tests for `getCommunity` and `getMemberships` because I have simplified the functions to only a wrapper around Clerk functions. And then the tests were only testing my Clerk mock.